catalog/video/catalog-mpegts-stream/src/main/java/org/codice/alliance/video/stream/mpegts/rollover/CatalogRolloverAction.java

```diff
@@ -172,19 +149,28 @@ private String generateFilename() {
 
     private void updateParentWithChildMetadata(Metacard childMetacard)
             throws RolloverActionException {
-        parentMetacardUpdater.update(parentMetacard, childMetacard);
-        UpdateRequest updateRequest = createUpdateRequest(parentMetacard.getId(), parentMetacard);
-        submitParentUpdateRequest(updateRequest);
+        if (context.getParentMetacard()
```

Comment:
the code throws exceptions, and you can't throw a checked exception from ifPresent(...)